src/librustdoc/core.rs

```diff
@@ -402,7 +402,9 @@ pub fn run_core(options: RustdocOptions) -> (clean::Crate, RenderInfo, RenderOpt
                 let hir = tcx.hir();
                 let body = hir.body(hir.body_owned_by(hir.as_local_hir_id(def_id)));
                 debug!("visiting body for {:?}", def_id);
-                EmitIgnoredResolutionErrors::new(tcx).visit_body(body);
+                tcx.sess.time("emit ignored resolution errors", || {
+                    EmitIgnoredResolutionErrors::new(tcx).visit_body(body);
+                });
```

Comment:
Note: not a bad change, but would not have been marked as `<unknown>` before (it would show up as part of `typeck`).